src/main/java/seedu/address/model/issue/RoomNumber.java

```diff
@@ -9,11 +9,10 @@
  */
 public class RoomNumber {
 
-    public static final String MESSAGE_CONSTRAINTS = "Room numbers be of the format ##-###[-a] "
-            + "where # is a number and 'a' is an alphabet.";
+    public static final String MESSAGE_CONSTRAINTS = "Room numbers should be formatted as such: XY-ABC, "
```

Comment:
Would it make sense to just reference the variables from the `room.RoomNumber` class instead?